python/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py

```diff
@@ -339,8 +340,8 @@ def optionsString(self, file=None):
                 sdatasetsnew += """ \"LFN:%s\",""" % f.lfn
                 sdatasetsold += """ \"DATAFILE='LFN:%s' %s\",""" % (f.lfn, dtype_str)
             else:
-                sdatasetsnew += """ \"PFN:%s\",""" % f.namePattern
-                sdatasetsold += """ \"DATAFILE='PFN:%s' %s\",""" % (f.namePattern, dtype_str)
+                sdatasetsnew += """ \"%s\",""" % f.accessURL()[0]
+                sdatasetsold += """ \"DATAFILE='%s' %s\",""" % (f.accessURL()[0], dtype_str)
```

Comment:
Does the `accessURL` work for all file types? (I think I remember finishing this but I'm not sure I ever tested it)